docs/reference/migration/migrate_5_0/scripting.asciidoc

```diff
@@ -1,6 +1,54 @@
 [[breaking_50_scripting]]
 === Script related changes
 
+==== Switched Default Language to Painless
+
+The default scripting language for Elasticsearch is now Painless.  Painless is a language similar to Groovy, except
+faster and more secure.  Many Groovy scripts will be identitical to Painless scripts to help make the transition
+between languages as simple as possible.
+
+Documentation for Painless can be found at
+https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-scripting-painless.html
```

Comment:
I think there's a way to do relative links so this doesn't break in the future if we change branch names, maybe @clintongormley knows how?